### DISCUSSION TITLE
Level up scroll

### DISCUSSION THREAD

**Discussion by Clownsonged** [runelite/runelite#16188] [Ideas]
Sorry if it has been asked a few times, but a level up scroll would be amazing as seen below, so much nosalgia and quite helpful for newer players. 
![Screenshot 2023-01-21 134650](https://user-images.githubusercontent.com/24254199/213869780-9109263d-2b4e-4983-99db-56877e331b0d.png)

**Comment by Ben-Colwell** [comment]
I'll see what I can do for this, no promises

**Reply by Clownsonged** [reply]
Amazing! Let me know if you make any progress :)
